src/data/formatters.test.js

```diff
@@ -3,8 +3,8 @@ import {
   formatDateRange,
   formatTime,
   formatPrice,
-  formattedEventPrice,
-  formattedEventPriceRange,
+  formattedShortEventPrice,
+  formattedLongEventPrice,
```

Comment:
I would rename these to follow the same tense as the other format functions. I.e. `formatShortEventPrice` and `formatLongEventPrice`.